torch/utils/mobile_optimizer.py

```diff
@@ -0,0 +1,22 @@
+"""
+This module contains utility method for mobile model optimization and lint.
+"""
+
+import torch
+
+
+def optimize_for_mobile(scripted_model):
+    """
+    Args:
+        scripted_model: An instance of torch script module with type of ScriptModule
+
+    Returns:
+        scripted_model: A new optimized torch script module, the method does not do
```

Comment:
You dont need to say, it doe not do inplace transformation. Given it already returns a new module, I think it is expected.